integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -726,6 +750,18 @@ void testCustomReadinessProbeFailureThresholdSuccessThreshold() {
             managedServerPodsCreationTime.get(managedServerPodName));
       }
     }
+
+    // check the admin server is up and running
+    checkPodReady(adminServerPodName, domainUid, domainNamespace);
+
+    // check the managed servers are up and running
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS_MIIDOMAIN; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;
```

Comment:
can you use checkPodReadyAndServiceExists
checkPodReadyAndServiceExists(managedServerPrefix, domainUid, domainNamespace);